{{cookiecutter.project_slug}}/webpack.config.js

```diff
@@ -1,20 +1,27 @@
 /* eslint-env node */
 const path = require('path');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const TerserJSPlugin = require('terser-webpack-plugin');
 const SpriteLoaderPlugin = require('svg-sprite-loader/plugin');
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
 
 module.exports = {
   mode: process.env.NODE_ENV,
+  devtool: 'inline-source-map',
   resolve: {
     modules: [
-      path.resolve(__dirname, 'assets/scripts'),
-      path.resolve(__dirname, 'assets'),
-      'node_modules',
+      path.resolve(__dirname, 'assets/'),
+      path.resolve(__dirname, 'assets/js/'),
+      path.resolve(__dirname, 'node_modules'),
```

Comment:
I'm in favor of removing the whole `modules` block, I think we should be explicit and always use the `@` alias. It's more efficient when resolving and easier to configure in the IDE.

About this, I'm in favor of introducing an extra `jsconfig.json` file to enable intellisense in VSCode and other editors:

```json
{
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "@/*": ["./assets/*"]
    }
  },
  "include": ["assets/**/*"]
}
```